peps/pep-8106.rst

```diff
@@ -0,0 +1,91 @@
+PEP: 8106
+Title: Enabling certificate verification by default for stdlib mail modules
+Version: $Revision$
+Last-Modified: $Date$
+Author: Martin Schobert <martin@pentagrid.ch>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 17-Nov-2023
+Sponsor: @vstinner 
+
+Abstract
```

Comment:
This section reads like a motivation, not an abstract.

Reply:
Renamed to 'Motivation'.